test/unit/services/igdbClient.spec.ts

```diff
@@ -0,0 +1,72 @@
+import * as igdbClient from '../../../src/services/igdbClient';
+import axios from 'axios';
+import { mocked } from 'jest-mock';
+
+jest.mock('axios');
+const mAxiosPost = mocked(axios.post);
+
+/**
+ * Tests associated with authenticate
+ */
+describe('authenticate test cases', () => {
+  const authToken = 'token';
+
+  afterEach(() => {
+    mAxiosPost.mockClear();
+  });
+
+  it('sucessfully returns a token', async () => {
+    mAxiosPost.mockResolvedValue({
+      data: {
+        access_token: 'token'
+      }
+    });
+    const result = await igdbClient.authenticate();
+    expect(mAxiosPost).toHaveBeenCalled();
+    expect(result).toBe(authToken);
```

Comment:
I might be being dumb but if we mock the return value to be an auth token, won't this always return the token? 

Out of the scope of this ticket but we'll probably want some integration tests to actually test the interaction with the API

Reply:
Yeah it will! This is mostly just testing that axios gets called and doesn't error. And 1000% to integration tests. I am keeping it simple now but we need them down the line (will be easier with some form of deployment)